docs/references.bib

```diff
@@ -132,6 +132,13 @@ @article{Hansen1974LightScatteringPlanetary
   journal      = {Space Science Reviews},
   doi          = {10.1007/BF00168069}
 }
+@article{Hapke1984BidirectionalReflectanceSpectroscopy},
+  title        = {Bidirectional Reflectance Spectroscopy; 3. Correcting for Macroscopic Roughness}
+  author       = {Bruce Hapke},
+  year         = 1984,
+  journal      = {Icarus}
+  doi          = {10.1016/0019-1035(84)90054-X}
```

Comment:
You're missing a few standard entries. Here is the BibTeX record exported by the journal:
```bibtex
@article{HAPKE198441,
title = {Bidirectional reflectance spectroscopy: 3. Correction for macroscopic roughness},
journal = {Icarus},
volume = {59},
number = {1},
pages = {41-59},
year = {1984},
issn = {0019-1035},
doi = {https://doi.org/10.1016/0019-1035(84)90054-X},
url = {https://www.sciencedirect.com/science/article/pii/001910358490054X},
author = {Hapke, Bruce},
}
```